WordPress/gutenberg
# Global Styles: Caption element UI controls

## What?

Add caption elements to the list of elements that can be customized in the Global Styles UI.
Closes https://github.com/WordPress/gutenberg/issues/44071

## Why?

So far, this could only be achieved via theme.json, now we can control these elements with Global Styles too.

## How?

Adding captions to the list of elements in the color and typography sections of Global Styles

## Testing Instructions





I'm using `emptytheme` with the following in theme.json:

```
	"styles": {
		"elements": {
			"caption": {
				"color": {
					"background": "red",
					"text": "yellow"
				},
				"typography": {
					"fontWeight": "bold",
					"fontStyle": "italic",
					"lineHeight": "2",
					"fontSize": "30px"
				}
			}
		}
	}

```

Go to global styles and change typography and color settings for the captions. Check that the frontend matches what you see on the editor.

## Screenshots or screencast 


https://user-images.githubusercontent.com/3593343/190431203-c11aaa48-38e9-4234-8d07-f008a6c99b4c.mp4
